sql/sqlcheck/sqlcheck_test.go

```diff
@@ -89,3 +122,35 @@ type testFile struct {
 func (t testFile) Name() string {
 	return t.name
 }
+
+type mockDriver struct {
```

Comment:
The mock does not need to have any logic in it. I should get the desired response as an argument and return it.